lib/pbench/cli/agent/commands/results/push.py

```diff
@@ -25,7 +25,7 @@ def execute(self) -> int:
         return 0
 
 
-@click.command()
+@click.command("pbench-results-push")
```

Comment:
I don't think this accomplishes anything useful.

That is, for external purposes, I think the command name is set in `setup.cfg`.  For local purposes, I expect that we're going to want the command name to be just `"push"`, because it will be a sub-command of the `"results"` subcommand.

Reply:
Without a `name` attribute `get_default_prog_name` raises an exception.

Reply:
This sounds like a symptom of another problem.  (What was the exception?)

Per the documentation, the [`name` attribute is optional](https://click.palletsprojects.com/en/8.0.x/api/#click.command), and [`get_default_prog_name`](https://click.palletsprojects.com/en/8.0.x/api/#click.testing.CliRunner.get_default_prog_name) should work if it is not supplied.

In any case, if you're going to supply an explicit name, here, I believe it should be just `"push"`, since ultimately this command will be part of the `pbench` tool in the `results` command group, and we don't want to end up with it being reported as `pbench-results-pbench-results-push`.

Reply:
This is not something we should resolve here because the general solution for our hierarchical commands is not in place.  For now, when these commands are invoked the usage string has to say something other than `main`.